firmware/config/engines/engines.mk

```diff
@@ -43,5 +43,6 @@ ENGINES_SRC_CPP = $(PROJECT_DIR)/config/engines/ford_aspire.cpp \
 	$(PROJECT_DIR)/config/engines/geo_storm.cpp \
 	$(PROJECT_DIR)/config/engines/zil130.cpp \
 	$(PROJECT_DIR)/config/engines/honda_600.cpp \
+	$(PROJECT_DIR)/config/engines/me7pnp.cpp \
```

Comment:
this should be taken care by https://github.com/rusefi/rusefi/commit/d8747359af182d28b02a5a69329d66b435caea14